### DISCUSSION TITLE
Basic raycaster example linked from Canvas API doc results in 404

### DISCUSSION THREAD

**Discussion by mjparme** [mdn/mdn-community#171] [Content] [answered]
If you go to:

https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Basic_usage

You will see Related Topics on the left. Under the Canvas API -> Examples you will see a link "A basic raycaster" which is a link to this page: https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/A_basic_ray-caster

Visiting that page results in a 404.

**Comment by teoli2003** [comment, marked as answer]
This example has been archived. You find the code and the live example in the museum: https://github.com/mdn/museum/tree/main/canvas-raycaster

We should remove the link, which I'm doing right now: [here is the PR](https://github.com/mdn/yari/pull/6845).
